web-marketplace/src/pages/Projects/AllProjects/AllProjects.tsx

```diff
@@ -265,7 +274,33 @@ export const AllProjects: React.FC<React.PropsWithChildren<unknown>> = () => {
               purchaseDetailsTitles={purchaseDetailsTitles}
               buttons={buttons}
               useProjectCardButton={IS_REGEN}
-              hasBottomCard={IS_REGEN}
+              creditIcon={
+                <>
+                  <InfoTooltip
+                    arrow
+                    placement="top"
+                    title={_(VOLUNTARY_MARKET_TOOLTIP)}
+                  >
+                    <img src={tebuLogo} alt="tebu" className="ml-3" />
+                  </InfoTooltip>
+                  {isComplianceProject && (
+                    <span className="flex items-center ml-5">
+                      {projectArea}
```

Comment:
this isn't necessary the project area that should be shown next to the "ha" logo on the project card but I assume the "credits available" amount as displayed on https://www.figma.com/design/brkxGV5qNOkZUp0YQl1cxO/Terrasos-Phase-1?node-id=661-81286&node-type=frame&t=LoNdXmJhR0ExHsds-0
could you confirm @erikalogie ?
this should be stored in sanity

Reply:
I added a new `complianceCredits` in Sanity project document.